tools/triage/add_triage_signature.py

```diff
@@ -2362,6 +2362,46 @@ def _process_ticket(self, url, issue_key):
             self._update_triaging_ticket("Controller logs contain some warnings:\n" + f"{{code}}{warning_text}{{code}}")
 
 
+class CustomerHasLoggedIntoCluster(Signature):
+    """
+    This signature will detect customer login to the cluster.
+    """
+
+    USER_LOGIN_PATTERN = re.compile(r"Accepted publickey for core from")
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            *args,
+            **kwargs,
+            comment_identifying_string="h1. Customer has logged into cluster nodes during installation",
+        )
+
+    def _process_ticket(self, url, issue_key):
+        md = get_metadata_json(url)
+
+        cluster = md["cluster"]
+        cluster_id = cluster["id"]
+
+        triage_logs_tar = get_triage_logs_tar(triage_url=url, cluster_id=cluster_id)
+
+        report = ""
+        for host in cluster["hosts"]:
+                    host_id = host["id"]
+
+                    try:
+                        journal_logs = get_host_log_file(triage_logs_tar, host_id, "journal.logs")
+                    except FileNotFoundError:
+                        continue
+                    
+                    if self.USER_LOGIN_PATTERN.findall(journal_logs):
+                        report += dedent(
+                            f"""
+                        h2. Host {host_id}: found evidence of a user login during installation"""
+                        )
```

Comment:
one less indentation level here

Reply:
I think it still applies. Indentation is too much here

Reply:
Done again, hopefully I got it this time

Reply:
nope. lint still fails on it:
```
./build/lib/tools/triage/add_triage_signature.py:2387:21: E117 over-indented
```

so just make sure lint check passes before merging

Reply:
suggested a code-change in another commit. you can also apply it from UI

Reply:
OK, finally passing the lint